src/test/java/net/sf/jabref/importer/fileformat/InspecImportTest.java

```diff
@@ -0,0 +1,178 @@
+package net.sf.jabref.importer.fileformat;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibtexEntry;
+import net.sf.jabref.model.entry.BibtexEntryTypes;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+
+/**
+ *
+ * This is the test class of the InspecImporter. It tests all 4 methods of the class.
```

Comment:
Yes, but only when it has more information as the class name. The class name already states that this class contains tests of the InspecImporter. Why should I duplicate this in a comment? Especially in a test class which always has the purpose to include tests for the methods of the class it is created to test. 

JavaDoc is especially useful for application code. But, then again, one should only add comments that have value, e.g., contain examples, metaphors, explanation on how to use this method and class, pre/post conditions, invariants, ...